### DISCUSSION TITLE
File name in `shell` command

### DISCUSSION THREAD

**Discussion by bfxfer** [helix-editor/helix#5408] [Q&A]
Is there a shortcut to specify the filename in the `shell` command? Something similar to [`%` in vim](https://vim.fandom.com/wiki/Get_the_name_of_the_current_file).

I use `pandoc` to convert `md` files to `html`, and in vim, I run the following that creates a HTML file from the current Markdown file:

```
:!pandoc --from=gfm --to=html --output=%.html %
```

I can do something like below in Helix:

```
:sh pandoc --from=gfm --to=html --output=<FILENAME>.html <FILENAME>
```

But, writing <FILENAME> every time is tedious. Wondering if there's a way to pass the filename to `shell` without having to write the filename.

**Comment by Ordoviz** [comment]
Not yet: #3134
